api/data_refinery_api/serializers.py

```diff
@@ -447,7 +447,7 @@ class DetailedExperimentSerializer(serializers.ModelSerializer):
     annotations = ExperimentAnnotationSerializer(many=True, source="experimentannotation_set")
     samples = DetailedExperimentSampleSerializer(many=True)
     sample_metadata = serializers.ReadOnlyField(source="sample_metadata_fields")
-    organisms = serializers.StringRelatedField(many=True, read_only=True)
+    organism_names = serializers.StringRelatedField(many=True, source="organisms", read_only=True)
```

Comment:
I would expect this endpoint to return the full Organism model instead of just the name.

Reply:
This would mean we would need to change the search document serializer to also return the object as well.

Reply:
https://github.com/AlexsLemonade/refinebio/issues/2055